fe/src/main/java/org/apache/doris/analysis/DateLiteral.java

```diff
@@ -29,16 +31,58 @@
 
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
+import org.joda.time.LocalDateTime;
+import org.joda.time.format.DateTimeFormatter;
+import org.joda.time.format.DateTimeFormatterBuilder;
 
 import java.io.DataInput;
 import java.io.DataOutput;
 import java.io.IOException;
 import java.nio.ByteBuffer;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
```

Comment:
And I see you use SimpleDateFormat and DateTimeFormatter.
Can we unify them to DateTimeFormatter?